docs/cluster_agent_setup.md

```diff
@@ -0,0 +1,69 @@
+# Cluster Agent
+
+To deploy the Cluster Agent along with the Agent update the current `DatadogAgent` resource with the [`datadog-agent-with-clusteragent.yaml` file](https://github.com/DataDog/datadog-operator/blob/master/examples/datadog-agent-with-clusteragent.yaml):
+
+```yaml
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog-agent
+spec:
+  # Credentials to communicate between:
+  #  * Agents and Datadog (API/APP key)
+  #  * Node Agent and Cluster Agent (Token)
+  credentials:
+    apiKey: <DATADOG_API_KEY>
+    appKey: <DATADOG_APP_KEY>
+    token: <DATADOG_CLUSTER_AGENT_TOKEN>
+
+  # Node Agent configuration
+  agent:
+    image:
+      name: 'datadog/agent:latest'
+    config:
+      tolerations:
+        - operator: Exists
+
+  # Cluster Agent configuration
+  clusterAgent:
+    image:
+      name: 'datadog/cluster-agent:latest'
+    config:
+      metricsProviderEnabled: true
+      clusterChecksEnabled: true
+    replicas: 2
+```
+
+**Note**: `<DATADOG_CLUSTER_AGENT_TOKEN>` is a custom 32 characters long token that you can defined, if omited, a random one is generated automatically.
+
+Then apply it with:
+
+```shell
+$ kubectl apply -n $DD_NAMESPACE -f datadog-agent-with-clusteragent.yaml
+datadogagent.datadoghq.com/datadog-agent configured
+```
+
+Verify that the Agent and Cluster Agent are correctly running:
+
+```shell
+$ kubectl get -n $DD_NAMESPACE dd datadog-agent
+NAME            ACTIVE   AGENT     CLUSTER-AGENT   AGE
+datadog-agent   True     Running   Running         15m22s
```

Comment:
We updated this, `kubectl get -n $DD_NAMESPACE dd datadog-agent` shows current state (Progressing, Running , Updating , Failed) with the values in the tuple representing (Requested / Ready / Uptodate)

```
NAME            ACTIVE   AGENT             CLUSTER-AGENT     CLUSTER-CHECKS-RUNNER   AGE
datadog-agent   True     Running (2/2/2)   Running (2/2/2)                           6m
```